### PR TITLE
feat: add depository get command

### PR DESCRIPTION
**Pull request by 0xff-dev**
feat: add depository get command
https://github.com/bestchains/bc-cli/issues/1

### CODE REVIEW

File: cmd/bc-cli/main.go [hunk 1]
```diff
@@ -25,7 +25,8 @@ import (
 func NewCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "bc-cli [usage]",
-		Short: "bc-cli",
+		Short: "Command line tools for Bestchains",
+		Long:  `bc-cli is a command tool for bestchain that can query and create a variety of block resources.`,
```
Comment (bjwswang) [inline cmd/bc-cli/main.go:29]: blockchain resources?

File: pkg/depository/get.go [hunk 2]
```diff
@@ -17,17 +17,144 @@ limitations under the License.
...
+	}
+	contentName, _ := cmd.Flags().GetString("contentName")
+	if contentName != "" {
+		query.Add("contentName", contentName)
+	}
+
+	return fmt.Sprintf("%s?%s", common.ListDepository, query.Encode())
+}
+
+var headers = []string{"index", "kid", "platform", "operator", "owner", "blockNumber", "time"}
```
Comment (bjwswang) [inline pkg/depository/get.go:59]: Make first charater upper case ? 
like 
```
Index Kid Platform Operator
```
At least, we should display in upper case

File: pkg/printer/i.go [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	"strings"
+	"text/tabwriter"
+)
+
+// Printer print the contents of the structure based on the given header information
+type Printer interface {
+	GetByHeader(string) string
+}
+
+func Print(outpu io.Writer, headers []string, objs []Printer) {
```
Comment (bjwswang) [inline pkg/printer/i.go:15]: for this kind of base library funcs, we need to add some unit tests

File: pkg/depository/get.go [hunk 4]
```diff
@@ -17,17 +17,144 @@ limitations under the License.
...
+	cmd.Flags().IntP("size", "s", 10, "pagination size")
+	cmd.Flags().StringP("kid", "k", "", "search depository by kid")
+	cmd.Flags().StringP("name", "n", "", "search depository by name")
+	cmd.Flags().StringP("contentName", "c", "", "search depository by content name")
+	cmd.Flags().StringP("host", "", "", "bc-saas server")
+
+	return cmd
+}
+
+func DoGetReq(u string, heades map[string]string) ([]byte, error) {
```
Comment (bjwswang) [inline pkg/depository/get.go:131]: As other blockchain resources also need do `Get`, it's better to put this into a common library or utilities.

BTW... post/delete as well

**Review by bjwswang — CHANGES_REQUESTED** [review]

**Review by bjwswang — APPROVED** [review]
/lgtm

**Review by Abirdcfly — APPROVED** [review]

**Pull request merged**